src/index.js

```diff
@@ -53,9 +62,8 @@ module.exports = (
     cache: cache === false ? undefined : {
       type: "filesystem",
       cacheDirectory: typeof cache === 'string' ? cache : nccCacheDir,
-      name: "ncc",
-      version: require('../package.json').version,
-      store: "instant"
+      name: `ncc_${hashOf(entry)}`,
```

Comment:
Should this not be a hash of the project directory, so that if the input changes the cache can still be used?